tools/docker/scripts/test_regtest.sh

```diff
@@ -19,6 +19,17 @@ if command -v ompi_info &> /dev/null ; then
     export OMPI_MCA_plm_rsh_agent=/bin/false
 fi
 
+# Switch to stable DBCSR version if requested.
+if [ -n "${USE_STABLE_DBCSR}" ] ; then
+    echo "Switching to stable DBCSR version..."
+    if ! git -C cp2k/exts/dbcsr checkout -b v2.1.0-rc16 6c52382 ; then
```

Comment:
you should be able to simply do `git checkout v2.1.0-rc16`, no?
And why is this not an `ARG`?

Reply:
I prefer to the SHA, tags can change.

I agree it's a bit hackish. A cleaner way would be to move DBCSR into the toolchain and add a `--use-stable-dbcsr` flag.

Reply:
DBCSR in the totalchain is expected for the next year... This is will be v3.x...

Reply:
> I prefer to the SHA, tags can change.

Can, yes, will, no. With DBCSR packaged in Spack (and others) we would break things if we would be start changing tags. Please use tags instead to make it more maintainable.

Reply:
Ok, I switch to using the tag.